src/main/cv/python/apriltagTest.py

```diff
@@ -0,0 +1,31 @@
+import cv2
+import numpy as np
+from apriltag import AprilTag
+from visionInput import VisionInput
+import time
+
+RES = (640, 480)
+
+tag_module = AprilTag()
+
+
+FOV = (50.28, 29.16)
+CAM_HEIGHT = 0.4
+CAM_ANGLE = -15
+input = VisionInput(FOV, RES, CAM_HEIGHT, CAM_ANGLE)
+TAG_LENGTH_METERS = 0.165
+
+while True:
+    frame = input.getFrame()
+    annotated_frame = frame.copy()
+    tagData = tag_module.estimate_3d_pose(frame, annotated_frame, TAG_LENGTH_METERS)
+    print(tagData)
+    pose_list = [4000 for _ in range(16 * 6)]
+    for key, value in tagData.items():
```

Comment:
You're packing the pose data into a dictionary in `estimate_3d_pose`, just to unpack it here. Why not just return a list of poses in the list you want?